src/main/java/io/stargate/sgv2/jsonapi/api/v1/metrics/JsonApiMetricsConfig.java

```diff
@@ -26,6 +26,18 @@ public interface JsonApiMetricsConfig {
   @WithDefault("sort.type")
   String sortType();
 
+  @NotBlank
+  @WithDefault("json.shredding.serialization.total.documents")
+  String jsonShreddingSerializedTotalDocuments();
+
+  @NotBlank
+  @WithDefault("json.shredding.serialization.size")
+  String jsonShreddingSerializedSize();
+
+  @NotBlank
+  @WithDefault("json.shredding.serialization.performance")
```

Comment:
I think "performance" is ambiguous. Could be "time.msecs" instead.

I'll ask @jeffreyscarpenter what he thinks would be nice and consistent naming for three things here (see above):

1. Number of documents serialized for operation (1 for insertOne(), 1-20 for insertMany())
2. Total serialized size of documents of operation (in Java chars)
3. Total time taken by serialization of all docs for operation

Reply:
> 1. Number of documents serialized for operation (1 for insertOne(), 1-20 for insertMany())
I'm ok with the current name `json.shredding.serialization.total.documents`, but would also be ok with `json.shredding.serialization.documents`.

> 2. Total serialized size of documents of operation (in Java chars)
I'm ok with the current name `json.shredding.serialization.size`

> 3. Total time taken by serialization of all docs for operation
For other histograms we use in Cassandra and Stargate, I see terms like "latency" or "duration" used frequently. So `json.shredding.serialization.duration` seems good.